File: Daily Test/Java/Pattern Printing - Mirror Image With Hyphens 13.05.2026.java
import java.util.*;
public class Hello {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        int dash = 1;
        for(int i = n; i >= 1; i--){
            for(int j = 1; j <= i; j++){
                System.out.print(i);
            }
            for(int k = 1; k <= dash; k++){
                System.out.print("-");
            }
            dash += 2;
            for(int j = 1; j <= i; j++){
                System.out.print(i);
            }
            System.out.println();
        }
    }
}

File: Daily Test/C/Pattern Printing - Mirror Image With Hyphens 13.05.2026.c
#include <stdio.h>
int main() {
    int n;
    scanf("%d", &n);
    int dash = 1;
    for(int i = n; i >= 1; i--) {
        for(int j = 1; j <= i; j++) {
            printf("%d", i);
        }
        for(int k = 1; k <= dash; k++) {
            printf("-");
        }
        dash += 2;
        for(int j = 1; j <= i; j++) {
            printf("%d", i);
        }
        printf("\n");
    }
    return 0;
}
